project/MollyBuild.scala

```diff
@@ -28,7 +27,6 @@ object MollyBuild extends Build {
     settings = buildSettings ++ Seq(
       scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature"),
       scalaVersion in scalaZ3 := "2.11.2",
-      scalaVersion in "bloom-compiler" := "2.10.3",
```

Comment:
This statement doesn't seem to have any effect and confused me, so I removed it.